index.bs

```diff
@@ -1072,12 +1200,30 @@ spec: workers; urlPrefix: https://www.w3.org/TR/workers/#
         significant.
       </p>
 
+      <p>
+        HDR detection is more nuanced. Adding colorGamut, transferFunction,
+        and hdrMetadataType has the potential to add significant entropy. However,
+        for UAs whose decoders are implemented in software and therefore whose
+        capabilities are fixed across devices, this feature adds no effective
+        entropy. Similarly, for UAs whose decoders are implemented by the platform,
```

Comment:
The last bit about platform decoders seems a bit off. I think what you might be trying to say is that folks generally fall into big buckets where everyone has roughly the same capabilities. Roughly: laptops and desktops, android phones, iphones, and TVs/dongles. But that last one (TVs) isn't quite right - TVs will have fixed capabilities, and some of them will have different fixed capabilities (e.g. supporting only HDR10+ or DV metadata) - and we are definitely exposing those differences.

Reply:
> TVs will have fixed capabilities, and some of them will have different fixed capabilities (e.g. supporting only HDR10+ or DV metadata) - and we are definitely exposing those differences.

You may be over-emphasising those differences, particularly given what is already in the HTTP User Agent string for a TV. Many HTTP User Agent strings for TV sets identify the manufacturer and the model number or product family - app providers want (insist) on this to allow them to work around bugs or to make it easier to reconcile problems detected by an app with complaints by consumers about A/V content being poor quality or breaking completely. If the HTTP UA identifiers the TV as (for example) a Philips OLED973 then all the HDR capabilties expose no extra entropy at all. Even if the HTTP UA doesn't identify the exact model but identifies the manufacturer, year and product family then there would likely not be any additional information from the HDR capabilities. Product family might be explicit or implicit based on OS (Android / Tizen / Linux ) or HTML UA version.